### ISSUE TITLE
Cursor problem

### ISSUE THREAD

**Issue by equeim** [Pauan/AutoScroll#13]
After I stop autoscrolling Chromium can't display pointer cursor over links on most websites unless I do left-click on empty space of the page.

**Comment by Pauan** [comment]
Yeah, I've noticed some weird stuff with the cursor not resetting.

I'll look into this, thanks for letting me know!
